03/styles/global.css

```diff
@@ -4,23 +4,88 @@
     box-sizing: border-box;
 }
 
-.p, .t, .v {
+.p,
+.t,
+.v {
     border: 3px solid transparent;
     min-height: 50px;
-
     display: flex;
     justify-content: center;
     align-items: center;
 }
 
 .p {
-    border-color: #ffffba
+    border-color: #ffffba;
+    text-align: center;
+    grid-area: picture;
+}
+
+.p picture img {
+    max-width: 99%;
+    max-height: 99%;
+    margin: 0 auto;
 }
 
 .t {
     border-color: #ffdfba;
+    grid-area: text;
 }
 
 .v {
+    height: 0;
+    padding-bottom: calc(315 / 560 * 100%);
+    position: relative;
+    width: 100%;
     border-color: #baffc9;
+    grid-area: video;
+}
+
+.v iframe {
+    position: absolute;
+    top: 0;
+    left: 0;
+    height: 100%;
+    width: 100%;
+}
+
+@media (min-width: 600px) and (max-width: 1119px) {
+    body {
+        display: grid;
+        grid-template-areas: "picture" "text" "video" "text";
```

Comment:
Podobny problem z cudzysłowem jak w poprzednim zdaniu.
Jak już poprawisz zadanie 2 myślę, że łatwiej będzie Ci zrobić to zadanie.